ui/store/actionConstants.ts

```diff
@@ -108,3 +108,7 @@ export const SET_CUSTOM_TOKEN_AMOUNT = 'SET_CUSTOM_TOKEN_AMOUNT';
 ///: BEGIN:ONLY_INCLUDE_IN(flask)
 export const FORCE_DISABLE_DESKTOP = 'FORCE_DISABLE_DESKTOP';
 ///: END:ONLY_INCLUDE_IN
+
+///: BEGIN:ONLY_INCLUDE_IN(mmi)
+export const SHOW_ACCOUNT_DETAIL = 'SHOW_ACCOUNT_DETAIL';
```

Comment:
This isn't necessary...unless i'm missing where this is handled. I removed this in #17512

Reply:
Removed from code, it was needed because of the wrongly imported again `showAccountDetail` method